src/miner_hbbft_sidecar.erl

```diff
@@ -146,6 +146,8 @@ handle_call({submit, Txn}, From,
                             case blockchain_txn:absorb(Txn, Chain) of
                                 ok ->
                                     spawn(fun() ->
+                                                  %% this will now return {ok, Position, Length}
```

Comment:
or just plain `ok` until the TODO in the handler is implemented? 

Separately, there are a few other cases in the sidecar where this is called which may be worth documenting as well.

Reply:
it it helps, i will be changing sidecar in another PR to use the responses....